### PR TITLE
Add operation name for other volume operations

### PR DESCRIPTION
**Pull request by cwdsuzhou**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>

/kind feature
/kind bug

**What this PR does / why we need it**:
Though, [this pr](https://github.com/kubernetes/kubernetes/pull/75213) allows different operation names of the same pod and volume, currently this is only effective for `mount, unmount`. Other volume operations' names have not been set.
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by cwdsuzhou** [comment]
@gnufied @jingxu97 @rootfs

PTAL

**Comment by cwdsuzhou** [comment]
ping @saad-ali for approve

File: pkg/volume/util/operationexecutor/operation_generator.go [hunk 1]
```diff
@@ -1163,6 +1168,7 @@ func (og *operationGenerator) GenerateUnmapVolumeFunc(
 	}
 
 	return volumetypes.GeneratedOperations{
+		OperationName:     "map_volume",
```
Comment (saad-ali) [inline pkg/volume/util/operationexecutor/operation_generator.go]: typo: `unmap_volume`

**Review by saad-ali — CHANGES_REQUESTED** [review]

File: pkg/volume/util/operationexecutor/operation_generator.go (hunk 1 above)
Comment (cwdsuzhou) [reply pkg/volume/util/operationexecutor/operation_generator.go]: my mistake, updated,  thanks

**Comment by cwdsuzhou** [comment]
ping @gnufied @saad-ali

PTAL again

**Comment by cwdsuzhou** [comment]
Thanks @gnufied , could you add LGTM since I have changed the PR

**Pull request merged**